src/commands/uploadMobile.js

```diff
@@ -0,0 +1,73 @@
+import { uploadMachO } from '../uploadMacho.js';
+import { uploadProguard } from '../uploadProguard.js';
+
+const PLATFORMS = {
+  IOS: 'ios',
+  ANDROID: 'android',
+};
+
+const PLATFORM_UPLOAD = {
+  [PLATFORMS.IOS]: uploadMachO,
+  [PLATFORMS.ANDROID]: uploadProguard,
+};
+
+export const command = 'upload-mobile <path(s..)>';
+export const describe = 'Upload debug files for a mobile release';
+
+const sampleiOSUploadPath = '/build/MyApp-iphoneos.xcarchive/dSYMs/MyApp.framework.dSYM/Contents/Resources/';
+const sampleAndroidUploadPath = '/app/build/outputs/mapping/debug/mapping.txt';
+const usageStr = `Usage: logrocket upload-mobile -r <release> -pf <platform> <path(s..)>
+For android, include one path directly to debug file, like \`${sampleAndroidUploadPath}\`
+See android dev docs for information on how to shrink and obfuscate your code https://developer.android.com/build/shrink-code#enable
```

Comment:
Again minor, but it seems a little redundant to have the android path example both here and in `demandPathStr`, when both will be shown any time the path is invalid (even if the user has specified ios as platform)- maybe this could just say something like "[...] where `<paths>` is the relative or absolute filepath to either a single ProGuard mapping file, or a directory containing debug symbol files" - thoughts?

Reply:
The demand string only displays if they attempt to execute the command without a path or paths. Do you think we should skip it in the main usage string (which I believe only displays when you run `logrocket upload-mobile --help`), or leave it out of the demand string?

Reply:
the `.usage()` string actually shows any time the command is called without fully valid input, regardless of the --help flag - I think we should limit the example filepath and docs link to when the path value is missing or invalid (and if possible, only for `--platform android`)